templates/rotor/deployment.yaml

```diff
@@ -72,7 +72,7 @@ spec:
             {{- toYaml . | nindent 12 }}
           {{- end }}
           image: ghcr.io/groundnuty/k8s-wait-for:v2.0
-          args: ["service", "{{ include "jitsu.fullname" . }}-kafka"]
+          args: ["service", "{{ .Release.Name }}-kafka"]
```

Comment:
If you look at other Services like Bulker and Ingest, they are also waiting for Release.Name, which makes sense because the Kafka chart is not part of the JITSU one:
https://github.com/stafftastic/jitsu-chart/blob/0a601f3507903088a3ba189a8da850833a106b23/templates/bulker/deployment.yaml#L75